public/index.html

```diff
@@ -41,17 +41,25 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
         <!-- index.normal-->
-        <link href="{{{DECORATOR_URL}}}/css/client.css" rel="stylesheet" />
+        <link
+            href="https://dekoratoren.dev.nav.no/css/client.css"
```

Comment:
Skal DEV-url for dekoratør være hardkodet her?

Reply:
Jeg kan se om vi kan hente den inn som en env-variabel som react forstår. Eks via en `.env`fil. Da står vi litt friere til å teste med ulike dekoratører lokalt også. :)

Reply:
.env og slikt skal være på plass nå. :)